assets/js/modules/analytics/common/account-create/create-account-field.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * CreateAccountField component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	Input,
+	TextField,
+} from '../../../../material-components';
+import classnames from 'classnames';
+
+export default function CreateAccountField( {
+	hasError,
+	value,
+	setValue,
+	name,
+	label,
+} ) {
+	if ( 'undefined' === typeof value ) {
+		return null;
+	}
```

Comment:
I would argue it makes sense to keep this, since the initial values of these fields are still partially sourced by API data, and that's when the selectors would return `undefined`, basically the field defaults are still loading. As soon as there is a value, the user's modifications should never bring the value back to an `undefined` state, even if the field is emptied it would be an empty string. So I think keeping this check in place to ensure the user can't touch the field before the default is there makes sense to me.

Reply:
When working on testing this I experienced a visual bug where the field labels were superimposed over the field values. Adding this check seemed to fix that bug:

![image](https://user-images.githubusercontent.com/2676022/81889148-16320f80-9560-11ea-9b27-c4520edc4fdc.png)

When debugging this I did verify that the component is rendered twice, once with the value unset and once with the value set. Somehow the first render triggered the bug (the fields normally show their label inside the field when the value is blank, then move to the top when a value is present).

That said, trying to reproduce this error now I am unable to :)

I feel like leaving the check is safe anyway, so I documented its purpose and checked to `value === undefined` in https://github.com/google/site-kit-wp/pull/1343/commits/b622dc3c7a307bbc966d821eebf95500cfe03d65.